addons/web/static/src/legacy/scss/bootstrap_overridden.scss

```diff
@@ -179,3 +179,9 @@ $breadcrumb-item-padding: .2em !default;
 // Code
 
 $code-color: $o-main-code-color!default;
+
+// User input typically entered via keyboard
+
+$kbd-color: $o-gray-700 !default;
+$kbd-bg: $o-gray-100 !default;
+$kbd-box-shadow: 0px 1px 1px rgba($o-black, 0.2), inset 0px -1px 1px 1px rgba($o-gray-200, 0.8), inset 0px 2px 0px 0px rgba($o-white, 0.8);
```

Comment:
Last thought: shouldn't we add `!default` to this line (like the other in this file) ?

Reply:
Done 👍